style.css

```diff
@@ -1,2 +1,188 @@
 @import url(./utility.css);
-@import url(./screens/css/login.css);
\ No newline at end of file
+@import url(./screens/css/login.css);
+
+.shop-btn{
+    background-color: transparent;
+    border: 2px solid var(--bg-color);
+    color: var(--white);
+  }
+  .home-content-title{
+    text-align: center;
+    padding: 2rem;
+  }
+  .home-img-div{
+      text-align: center;
+      margin: 1rem 10rem 0 10rem;
+      height:25rem;
+  
+      /* position: relative;   c-util pos-rel */
+    }
+    .home-img{
+  
+  
+      /* width: 100%;  e-util wd-100 */
+      /* height: 100%; this doc */
+      
+    
+    }
+  
+    .home-img-txt{
+      color: var(--white);
+  
+      /* position: absolute; c-util pos-rel */
+      
+      top: 10rem;
+      left: 5rem;
+    }
+  
+  
+  /* -------------------------home products------------------ */
+  
+    .home-product{
+      /* display: flex;
+      justify-content: center;
+      align-items: center; */
+  
+      gap: 10rem;
+      
+      /* width: 100%; e-util pos-rel */
+  
+      height: 10rem;
+     
+      
+    }
+  
+  .product-link a{
+    color: var(--text-color);
+  
+      font-size: 1.5rem;
+      display: flex;
+   
+      text-align: center;
+   
+      flex-direction: column; 
+    }
+  
+  .fontsize-lg{
+    font-size: 2rem ;
+  }
+  
+  /* -----------------------------home ad ---------------------------*/
+  
+  
+  .home-ad{
+    /* width: 100%; e-util pos-rel */
+  
+    height: 30rem;
+  
+    background-color: var(--bg-color);
+    color: var(--text-color);
+  
+    /* display: flex;
+    justify-content: center;
+    align-items: center;
+   */
+    gap: 20rem;
+  }
+  
+  
+/*   
+  .height-100{
+    height: 100%; 
+  } */
+  
+  .home-ad-img{
+  
+  width: 80%;
+  height: 80%;
+  
+  
+  }
+  
+  
+  
+  
+  /*-------------------- new arrivals -----------------------*/
+  
+  
+  .home-new-arrival{
+  
+  
+  
+    display: grid;
+    grid-template-columns: 20rem 20rem;
+    grid-template-rows: 25rem 25rem;
+    text-align: center;
+    justify-content: center;
+    row-gap: 2rem;
+    column-gap: 2rem;
+  
+    
+  }
+  
+  .home-new-arrival  div{
+   
+    border-radius: 4px;
+    box-shadow:  0 4px 8px 0 rgba(0, 0, 0, 0.2); 
+    background-color: var(--bg-color);
+  
+  }
+  
+  .arrival-head{
+    text-align: center;
+    font-size: 1.2rem;
+    color: var(--text-color);
+  }
+  
+  .arrival-img{
+  
+    /* width: 50%; */
+  
+  }
+  .arrival-link , .arrival-link a{
+    font-size: 1.2rem;
+    padding-top: 0.5rem;
+    font-weight: 900;
+    color: var(--text-color);
+  
+  }
+  
+  /* .padd-top-lg{
+    padding-top: 1.5rem;
+  }
+  .padd-btm-lg{
+    padding-bottom: 1.5rem;
+  }
+  .padd-top-sm{
+    padding-top: 0.5rem;
+  }
+  
+  .padd-md{
+  padding: 1rem;
+  } */
```

Comment:
If you dont want to use this in fiture you can remove it

Reply:
ooh i missed , thanks for pointing out